### PR TITLE
A Whole Bunch of Field Definitions

### PR DESCRIPTION
**Pull request by miriamseely**
This pull request includes field definitions for the following layers:

- Basic Parcels
- Counties
- Lakes
- Streams
- Land Ownership
- VISTA Ballot Areas

These are all the layers that I've worked on so far that have their own schema summary documents (in the future, I'll be including field definition md files with the pull request for the layer itself).  I filled in the definitions to the best of my knowledge, but if there are gaps or corrections that need to be addressed, let me know and I'll update these accordingly.

### CODE REVIEW

File: metadata/schema/counties_schema.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The Federal Information Processing Standard (FIPS) code for each county. A full list of county and state codes is available on the Federal Communications Commission’s [website](https://transition.fcc.gov/oet/info/maps/census/fips/fips.txt).
+
+### STATEPLANE
+
+Whether the county is a part of the North, Central, or South [State Plane Coordinate System](https://www.usgs.gov/faqs/what-state-plane-coordinate-system-can-gps-provide-coordinates-these-values) (SPCS) for Utah.
+
+### POP_LASTCENSUS
+
+The county population as of the 2020 census.
```
Comment (steveoh) [inline metadata/schema/counties_schema.md:35]: I recommend removing the date

File: metadata/schema/counties_schema.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### POP_CURRENTESTIMATE
+
+The current population estimate for the county.
```
Comment (steveoh) [inline metadata/schema/counties_schema.md:39]: We discovered when this was updated right? I'd explain how often it changes.

File: metadata/schema/counties_schema.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### GLOBAL_ID
+
+<!--- No definition for this field. -->
+
+### FIPS_STR
```
Comment (steveoh) [inline metadata/schema/counties_schema.md:45]: https://en.m.wikipedia.org/wiki/Federal_Information_Processing_Standards as a string value

File: metadata/schema/parcels_schema.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- Private
+- Tribal
+
+### RECORDER
+
+Phone number for the county recorders office for a given parcel.
+
+### ParcelsCur
+
+<!--- No definition for this field. -->
```
Comment (steveoh) [inline metadata/schema/parcels_schema.md:42]: This is the currency date the parcel was updated

File: metadata/schema/parcels_schema.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+### ParcelsRec
```
Comment (steveoh) [inline metadata/schema/parcels_schema.md:44]: This is the date the parcel data was received

File: metadata/schema/streams_schema.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+<!--- No definition for this field. -->
+
+### fdate
+
+The date the feature was added to the dataset.
+
+### resolution
+
+<!--- No definition for this field. -->
```
Comment (steveoh) [inline metadata/schema/streams_schema.md:19]: This has to do with scale. Like 1:24,000

Comment (miriamseely) [reply metadata/schema/streams_schema.md:19]: In the feature class, the resolution field is set to "2" for each of the features. Do we know what the 2 refers to? I was thinking it had to do with this being the second iteration of the NHD (which is high-resolution, whereas the first one I believe was 1:100,000).

Comment (steveoh) [reply metadata/schema/streams_schema.md:19]: @ZachBeck do you know?

Comment (ZachBeck) [reply metadata/schema/streams_schema.md:19]: It's a coded domain, 2 = High

**Pull request merged**